Libraries/Components/AccessibilityInfo/NativeAccessibilityInfo.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+export interface Spec extends TurboModule {
+  +isReduceMotionEnabled: (
+    resolve: (isReduceMotionEnabled: boolean) => void,
```

Comment:
I think a more appropriate name for this would be `onSuccess`. The name `resolve` only makes sense if you're working with promises.